script/Ciel/Day 7/07_01_CIEL07_6.txt

```diff
@@ -607,157 +625,163 @@ Arcueid?!"
 -- Page 545, Offset 16115.
 -- 俺は完全に不能だった。恐怖がないだけで、何の対策も考えていなかった。
-There is nothing I can do.
- Just because I wasn't scared, didn't mean I could just go in without a plan.
+There's nothing I can do.
+ Because I wasn't afraid, I came in here without coming up with a plan first.
 }
 [sha:f6bd1670ddbf9f1731f59b44474d559061caa9db]{
 -- Page 545, Offset 16116.
 -- だから死ぬ。ここで死ぬ。あの半ば腐った死体の爪が、豆腐をべちゃつかせるように、俺の頭をすくい取る。
-Which is why I will die――here.
- The claws of those half-rotted corpses will scoop out the contents of my head, turning to it mush.
+I'm dead.
+ I'm going to die here.
+ The claws of those half-rotted corpses will scoop out the contents of my head as easily as they'd mutilate a block of tofu.
 }
-[sha:11985ebf77373c0bdd7b8ea344918de6a6f01867]{
+[offset:16117]{
 -- Page 546, Offset 16117.
 -- 「―――、ァ―――」
-"Ah―――"
+"%{i}Ah%{/i}――%{i}haah%{/i}―――"
 }
 [sha:68ee2c815a0579675d1c366af18104f2269832dd]{
 -- Page 546, Offset 16118.
 -- 俺は完全に不能だった。一度窮地を経験したぐらいで<驕|おご>った理性が、遠野志貴である事を放棄した。
-There is nothing I can do.
- My mind, conceited at having experienced this predicament just once, gives up on being Tohno Shiki.
+There's nothing I can do.
+ My sense of reason, which had grown conceited after making it out of a predicament once before, gives up on being Tohno Shiki.
 }
 [sha:7521c480d9dc4335133bf2b85a2bb234372fbdde]{
 -- Page 547, Offset 16119.
 -- 「は―――ハ」
-"Hahhh――hah"
+"%{i}Hahhh――hah%{/i}"
 }
 [offset:16120]{
 -- Page 547, Offset 16120.
 -- だから。
-Which is why―――
+With it gone―――
 }
 [sha:ab1fd004aa26c0959cbd7105c7f0df95474ea6d0]{
 -- Page 547, Offset 16121.
 -- その後に残ったものは、<寄|ツ><せ|ギ><集|ハ><め|ギ>の人間性からやっと解放された、剥き出しの<己|おのれ>だった。
-That which then remains is my bare self,
- finally released from the humanity it had cobbled together.
+// threadbare/patchwork reincluded here for ツギハギ to hearken back to 00_00
+All that remains is my bare self,
+ finally released from the threadbare patchwork of humanity it had cobbled together.
 }
 [sha:d93bfba124b581da2e0dd206815dfbfebcafa299]{
 -- Page 548, Offset 16122.
 -- 意識より先に体が走る。
-My body moves before my consciousness can.
+My body moves before my mind has a chance to catch up.
 }
 [sha:b245d1754fbf8656e8faafa029a092d779c3704f]{
 -- Page 548, Offset 16123.
 -- わずかな戸惑いもなく解体する。血の匂いを、手触りのない肉の<断触|だんしょく>を記憶する。
-// ebin nasu pun <断触|だんしょく>
-// after this line, the visuals show Shiki remembering cutting Arcueid apart.
-I commence the dismemberment without an inkling of hesitation.
- I recall the smell of blood――the warm cuts of her untouched flesh.
+// P: ebin nasu pun <断触|だんしょく>
+// P: after this line, the visuals show Shiki remembering cutting Arcueid apart.
+// R: Just to make sure I understand this, the pun is <feeling of cutting|warm colour>?
+// R: Where the warm colour is of course more blood?
+I don't even so much as hesitate as I dismantle them.
+ I recall the smell of blood, the feeling of slicing through her untouched flesh, and the warm color that spilled from it.
```

Comment:
Yeah, I figured 断触 was a pun on 暖色. That said, the line as it is here seems a bit inelegant. Rather than try and preserve  all the semantic meaning, I feel like keeping to the tone of the prose is much better. I'd rather revert here tbh.

Reply:
Not sure I agree on elegance or reverting exactly. I don't feel the old line conveys to the EN reader either the sensation of cutting or the meaning of the warmth. 'Warm cuts' without the JP context sounds like you've come back from the butchers with a leg of lamb and are commenting on its temperature, a warm 'cut of meat' (noun) rather than a 'cutting of the meat' (verb). If you don't think it's worth having the final clause we can drop it, but I don't feel it harms the flow of the sentence.

Reply:
Maybe just move warm to blood and remove the last clause.

> I recall the warm smell of blood, the feeling of slicing through her untouched flesh.